app/controllers/draft_events_controller.rb

```diff
@@ -32,4 +32,42 @@ def join_draft
     }
     broadcast_message :join_draft, message, namespace: :drafts
   end
+
+  def pick_contestant
+    puts active_player == current_user
+    if active_player == current_user
+      puts 'in the right spot'
+      puts message
+      puts active_player.id
+      pick = Pick.new
+      pick.draft_id = message["draft_id"]
+      pick.contestant_id = message["contestant_id"]
+      pick.type = message["type"]
+      puts " PICK #{pick}"
+      _message = {
+          pick: pick,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.